pkg/metadata/names.go

```diff
@@ -0,0 +1,10 @@
+package metadata
+
+const (
+	// PluginName hold a name of the plugin.
+	PluginName = "kn-event"
+	// PluginUse hold a the plugin short name, intended to use as "kn <use>".
+	PluginUse = "event"
+	// PluginDescription holds a short description of the plugin.
+	PluginDescription = "A plugin for operating on CloudEvents"
```

Comment:
Wdyt about the following? Inspired by the readme and it's more aligned with rest of plugins. I can't decide which is better. :) 

I'll lgtm the PR and add hold for you decide to proceed with current description. 

```suggestion
	PluginDescription = "Manage CloudEvents from command line"
```

Reply:
I like better what you proposed 😊

Reply:
/unhold